### PR TITLE
fix(web-github): log JSON parse failures in fetch-prs, fetch-pr-body, fetch-workflow-runs

### PR DESCRIPTION
**Pull request by H31MDALLR**
Malformed JSON from successful `gh` commands previously made the web dashboard silently show empty or missing data. The three JSON parse boundaries now write contextual warnings to stderr and retain their parse-error fallback values: `[]` for PRs/runs and `nil` for PR details.

Warnings use the shared message translator and CLI system catalog (`:log/...`), as required by miniforge-standards rule 050. PR analysis runs outside the parsing catch: analysis exceptions now propagate instead of being swallowed or mislabeled as JSON failures. This is an intentional behavior change distinct from the unchanged malformed-JSON fallbacks.

Validation: 9 targeted tests / 24 assertions cover malformed JSON, stderr context, command failures, and analysis-exception propagation. Full pre-commit validation passes: Polylith structure, Clojure/stratum lint, 347 smoke tests / 1,310 assertions, and 8 GraalVM/Babashka compatibility tests / 667 assertions. The final diff received independent adversarial review.

### CODE REVIEW

File: bases/cli/src/ai/miniforge/cli/web/github.clj [hunk 1]
```diff
@@ -53,7 +53,12 @@
       (try
         (->> (json/parse-string (:out result) true)
              (mapv #(assoc % :repo repo :analysis (risk/analyze-pr %))))
-        (catch Exception _ [])))))
+        (catch Exception e
+          ;; gh succeeded (exit 0) but returned unparseable output — log
+          ;; so operators can diagnose format changes or partial writes.
+          (binding [*out* *err*]
+            (println (str "[warn] fetch-prs: JSON parse failed for repo " repo ": " (ex-message e))))
```
[the remark this answers is not in the document]
Comment (H31MDALLR) [reply bases/cli/src/ai/miniforge/cli/web/github.clj]: Fixed in 8cc750c. Narrowed the `try` to cover only `json/parse-string`; moved the `mapv`/`risk/analyze-pr` call outside it. A parse failure still returns `[]` (preserving caller semantics); an analysis exception now propagates rather than being silently mislabelled.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by H31MDALLR — COMMENTED** [review]
Standards/adversarial review at `8cc750ca9`: narrowing the parse try fixes the misleading parse diagnostic, but the three new `[warn] ... JSON parse failed ...` strings violate `miniforge-standards/foundations/localization.mdc` (050). Developer-facing prose must also be catalog-backed, in the system locale, through a translator. Extract a shared diagnostic helper, retain repository/PR context, and add focused tests capturing stderr for malformed JSON and asserting unchanged []/nil fallback values. No tests have yet changed for this new observable behavior (716).

Also document and test the deliberate behavior change in `fetch-prs`: `risk/analyze-pr` failures now propagate rather than returning [] as before. The current PR body says fallback contracts are unchanged, which is no longer true after the review fix. Test malformed JSON separately from an injected analysis failure so these paths cannot be conflated again (720).

**Comment by H31MDALLR** [comment]
Addressed in 2b8d824:

**Localization (050)** — three new catalog keys added to `en-US.edn` under `:web/github-fetch-*-json-failed`; `github.clj` now requires `messages` and calls `messages/t` with `{repo}`, `{number}`, `{cause}` params for all three diagnostic paths.

**Tests (716 / 720)** — new `github_test.clj` with 10 focused tests:
- Malformed JSON path for each of `fetch-prs`, `fetch-pr-body`, `fetch-workflow-runs`: asserts the safe fallback value (`[]` / `nil`) and that stderr carries the catalog warning with repo/number context.
- gh CLI failure for `fetch-prs` (→ `nil`) and `fetch-workflow-runs` (→ `[]`) assert blank stderr.
- `fetch-prs` analysis failure: asserts the exception propagates (`thrown-with-msg?`), keeping the JSON-parse and analysis-failure paths permanently distinct.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: bases/cli/src/ai/miniforge/cli/web/github.clj [hunk 2]
```diff
@@ -21,6 +21,7 @@
    [babashka.process :as process]
    [clojure.string :as str]
    [cheshire.core :as json]
+   [ai.miniforge.cli.messages :as messages]
```
[the remark this answers is not in the document]
Comment (H31MDALLR) [reply bases/cli/src/ai/miniforge/cli/web/github.clj]: Strata annotations are file-local (dewey 001/210): `{:stratum 2}` on `messages/t` describes its relative position within `messages.clj`'s own abstraction DAG — it sits above `catalog` within that file. The annotation carries no cross-file restriction on callers. `lint:stratum` confirmed this: the Lint check passed clean on commit `2b8d824` with no violations flagged. There is no upward stratum dependency here.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by H31MDALLR — COMMENTED** [review]
Standards follow-up at `2b8d824f1`: catalog-backed strings and diagnostic tests are now present. The remaining localization mismatch is the catalog location: these operational stderr warnings were placed in `messages/en-US.edn` under `:web/...`; rule 050 explicitly routes developer-facing logs to `messages/system.edn` (typically `:log/...`) via a system translator. Please use that system catalog rather than expanding user-localized UI copy with diagnostics. A small shared warning helper also avoids the repeated binding/println boilerplate.

**Review by H31MDALLR — COMMENTED** [review]
The remaining rule-050 gap is fixed in 3a3079349: all three operational warnings now use the shared translator with `config/cli/messages/system.edn` and `:log/...` keys. Targeted diagnostics tests pass (9 tests / 24 assertions). Mandatory pre-commit gates and commit hooks passed without bypass: structure, lint, stratum lint, 347 smoke tests / 1,310 assertions, and 8 compatibility tests / 667 assertions. I reviewed the final diff and updated the PR description to disclose analysis-exception propagation separately from malformed-JSON fallbacks. Fresh CI is running.
